app/src/main/res/values/strings.xml

```diff
@@ -256,17 +256,22 @@
     <string name="no_device_found">No devices found</string>
     <string name="setup_tut">If you\'re setting up a new device, make sure it\'s \n nearby and plugged into a wall outlet</string>
     <!--About Us String-->
+    <string name="susi_about_url">https://chat.susi.ai/overview</string>
     <string name="susi_about">
         <a href="https://chat.susi.ai/overview">SUSI AI</a> is an intelligent libre software personal assistant. It is capable of chat and voice interaction by using APIs to perform actions such as music playback, making to-do lists, setting alarms, streaming podcasts, playing audiobooks, and providing weather, traffic, and other real time information. Additional functionalities can be added as console services using external APIs. Susi AI is able to answer questions and depending on the context will ask for additional information in order to perform the desired outcome. The core of the assistant is the Susi AI server that holds the "intelligence" and "personality" of SUSI AI.</string>
     <string name="susi_contributors">Contributors</string>
+    <string name="susi_contributors_url">https://github.com/fossasia/susi_android/graphs/contributors</string>
     <string name="susi_contributors_desc">Developed by <a href="https://github.com/fossasia/susi_android/graphs/contributors">Contributors</a>.</string>
     <string name="susi_license_information">License</string>
+    <string name="susi_license_url">https://github.com/fossasia/susi_android/blob/master/LICENSE</string>
     <string name="susi_license_information_desc">This project is currently licensed under the Apache License Version 2.0. As per the <a href="https://github.com/fossasia/susi_android/blob/master/LICENSE">LICENSE.md</a>
     </string>
     <string name="susi_report_issues">Report Issues</string>
+    <string name="susi_report_issueurl">https://github.com/fossasia/susi_android/issues</string>
```

Comment:
issueurl --> issue_url
In my opinion it would be better to name these resources as url_susi_about, url_susi_contributors and so on. This makes it quicker to understand that it is a string resource for a url.